app/exec/extension/_lib/vsix-manifest-builder.ts

```diff
@@ -320,6 +320,11 @@ export class VsixManifestBuilder extends ManifestBuilder {
 					});
 				}
 				break;
+			case "githubflavoredmarkdown":
+				if(value !== true && value !== false) {
+					throw "Value for gitHubFlavoredMarkdown is invalid. Valid values are true or false.";
+				}
+				this.addProperty("Microsoft.VisualStudio.Services.GitHubFlavoredMarkdown", value);

```

Comment:
this.addProperty expects two strings, but your second parameter is a Boolean. This would be flagged as a compile-time error in TypeScript 2.0.